src/app/shared/components/dark-mode-toggle/dark-mode-toggle.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { MediaMatcher } from '@angular/cdk/layout';
+import { StorageService } from '../../services/storage.service';
+
+@Component({
+  selector: 'app-dark-mode-toggle',
+  templateUrl: './dark-mode-toggle.component.html',
+  styleUrls: ['./dark-mode-toggle.component.scss'],
+})
+export class DarkModeToggleComponent implements OnInit {
+  isDarkMode: boolean;
+
+  constructor(
+    @Inject(DOCUMENT)
+    private readonly document: Document,
+    private readonly mediaMatcher: MediaMatcher,
+    private readonly storageService: StorageService,
+  ) {}
+
+  ngOnInit() {
+    const userPrefersDark =
```

Comment:
Let's use the current (light) theme by default, regardless of the media value

Reply:
Done! ✅ ,now by default is in light mode.

link-preview: https://deploy-preview-1138--docs-nestjs.netlify.app